config/checkstyle_input_checks.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+        "https://checkstyle.org/dtds/configuration_1_3.dtd">
+
+<module name="Checker">
+    <property name="charset" value="UTF-8"/>
+    <property name="haltOnException" value="false"/>
+
+    <property name="cacheFile" value="target/cache_resources"/>
+
+    <!-- Suppressions for resources -->
+    <module name="SuppressionFilter">
+        <property name="file"
+                  value="${project.basedir}/config/checkstyle_input_suppressions.xml"/>
+    </module>
+
+    <!-- Check to make sure check UT input files have config and trailing comments -->
+    <module name="RegexpMultiline">
+        <property name="format"
+                  value="(\/\/ config:)"/>
```

Comment:
I do not see any Input file that follow this format,
Please do, following search regexp: 
```
~/java/github/romani/checkstyle/src/test/resources [master|✔] 
$ ag "\* Config" | grep java | wc -l
36
```

and remove 36 Inputs from suppression file.

There might be that some  Inputs missing `// ok` comment, please add it.

```
~/java/github/romani/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/checks [master|✔] 
$ ag "\* Config" | grep java | cut -d ':' -f 1
regexp/regexpmultiline/InputRegexpMultilineMultilineSupport2.java
naming/abbreviationaswordinname/InputAbbreviationAsWordInNameIgnoreNonStaticFinal.java
naming/abbreviationaswordinname/InputAbbreviationAsWordInNameIgnoreFinalKeepStaticFinal.java
naming/abbreviationaswordinname/InputAbbreviationAsWordInNameIgnoreFinal.java
naming/abbreviationaswordinname/InputAbbreviationAsWordInNameNoIgnore.java
naming/abbreviationaswordinname/InputAbbreviationAsWordInNameIgnoreStaticFinal.java
naming/abbreviationaswordinname/InputAbbreviationAsWordInNameIgnoreStatic.java
naming/abbreviationaswordinname/InputAbbreviationAsWordInNameIgnore.java
naming/abbreviationaswordinname/InputAbbreviationAsWordInNameIgnoreStaticKeepStaticFinal.java
regexp/regexpmultiline/InputRegexpMultilineMultilineSupport.java
coding/unnecessarysemicolonafteroutertypedeclaration/InputUnnecessarySemicolonAfterOuterTypeDeclaration.java
coding/fallthrough/InputFallThroughDefault.java
coding/avoiddoublebraceinitialization/InputAvoidDoubleBraceInitialization.java
coding/unnecessarysemicolonaftertypememberdeclaration/InputUnnecessarySemicolonAfterTypeMemberDeclaration.java
coding/arraytrailingcomma/InputArrayTrailingCommaAlwaysDemandTrailingComma.java
coding/requirethis/InputRequireThisExpressions.java
coding/avoidnoargumentsuperconstructorcall/InputAvoidNoArgumentSuperConstructorCall.java
blocks/rightcurly/InputRightCurlyAloneNewTokens.java
blocks/rightcurly/InputRightCurlyAloneBlocksWithSemi.java
blocks/rightcurly/InputRightCurlySameNewTokens.java
blocks/rightcurly/InputRightCurlySameBlocksWithSemi.java
blocks/rightcurly/InputRightCurlyAloneOrSingleLineNewTokens.java
blocks/rightcurly/InputRightCurlyAloneOrSingleLineBlocksWithSemi.java
blocks/needbraces/InputNeedBracesCaseDefaultNoSingleLine.java
imports/customimportorder/InputCustomImportOrderEmptyRule.java
imports/customimportorder/InputCustomImportOrder_OverlappingPatterns.java
imports/customimportorder/InputCustomImportOrderDefault2.java
whitespace/emptylineseparator/InputEmptyLineSeparatorSingleCommentUnderPackage.java
whitespace/emptylineseparator/InputEmptyLineSeparatorBlockCommentSeparatedFromPackage.java
whitespace/emptylineseparator/InputEmptyLineSeparatorClassDefinitionAndCommentNotSeparatedFromPackage.java
whitespace/emptylineseparator/InputEmptyLineSeparatorSingleCommentSeparatedFromPackage.java
whitespace/emptylineseparator/InputEmptyLineSeparatorBlockCommentUnderPackage.java
whitespace/emptylineseparator/InputEmptyLineSeparatorModifierUnderPackage.java
javadoc/missingjavadocmethod/InputMissingJavadocMethodPublicMethods.java
javadoc/javadoctagcontinuationindentation/InputJavadocTagContinuationIndentationDescription.java
javadoc/javadocmethod/InputJavadocMethodIgnoreThrows.java
```

Reply:
From this: https://github.com/checkstyle/checkstyle/pull/8413#discussion_r452872043
I thought we wanted `// config:`.  But  most comments regarding config are javadoc comments. I will update all config comments in other PRs to be javadoc comments, and change this regex to reflect this.

Reply:
yes, it was my fault, as we do this rarely ... we do not even remember what format we use :).